packages/backend/src/studio-api-impl.ts

```diff
@@ -235,6 +237,15 @@ export class StudioApiImpl implements StudioAPI {
     return podmanDesktopApi.navigation.navigateToPod(pod.kind, pod.Name, pod.engineId);
   }
 
+  async navigateToResources(): Promise<void> {
+    return podmanDesktopApi.navigation.navigateToResources();
+  }
+
+  async navigateToEditConnectionProvider(connectionName: string): Promise<void> {
+    const connection = getPodmanConnection(connectionName);
+    return podmanDesktopApi.navigation.navigateToEditProviderContainerConnection(connection);
```

Comment:
This API would not be available in version prior to 1.10.0, since we are supporting version from 1.8.0 we need to have a mechanism to not show the redirect button I guess ?

Reply:
Yes, i'm going to hide the button. Still in progress